section3/reflection.md

```diff
@@ -1,13 +1,27 @@
+- I also found myself taking more notes along the way, over-explaining myself in my notes when jotting down new concepts and terms. This sort of forced me to think ahead in a lot of ways, which made the exercises less scary when they were more challenging without instructions.
 
-2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+### 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+- Target App: listing items categorized by a specific department
+  - for example, when you click on the Baby Department you'll then see diapers, wipes, strollers, etc. listed under that department on the website
 
-3. In your own words, explain what an Array method is.
+### 3. In your own words, explain what an Array method is.
+- an `array` is the list of items under a single named variable
+  - for example: `var flowers = ['sunflower', 'peony', 'lavender']`
+- a method of listing which uses multiple items for one variables
+  - a way of grouping elements together
 
-4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+### 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+- The purpose of a loop is to simplify a code block, when you want to repeat the same action several times within the code.
+- The loop allows coders to avoid writing out a line of code over and over again, and it also lessens the length of the code entirely, making it simpler to read as well.
+- We would use a loop without an array when we would like to create a code that will return a different response to specified conditional if-statements. We use loops with an array when we would like to run a code that will have the same basic structure to it, only responding to the code that returns a different value within that structure.
 
-5. If you had to teach a 5 year old about loops, how would you explain it?
+### 5. If you had to teach a 5 year old about loops, how would you explain it?
+- Loops are like the automatic hand driers in the bathrooms you use after you wash your hands. The drier is set up in a way that it is always ready to blow air out. Every time your hands go into the drier, you can feel the air start to push out onto your hands. When there is no movement in the drier for a little bit, then it stops until the next time there are hands in the drier.
 
-6. What questions do you still have about Arrays and loops?
+### 6. What questions do you still have about Arrays and loops?
+- I'm curious of how to utilize arrays and loops on a deeper level. For example, how would a more complex app such as Amazon or Spotify use these methods? What would they look like? Are there several arrays that are very similar with subtle changes?
```

Comment:
As a FE developer, a lot of your job will be to take data, loop over it, and get it to show up in the browser. 

Let's say you are working for Spotify building a new page for an "Album" - including many songs, in most cases.
- You'd get an array of songs
- You'd loop over that array, and write one code block to instruct the browser to display the current song in a certain way